src/h5_async_vol.c

```diff
@@ -105,7 +105,7 @@ FILE *fout_g;
 /* The async VOL wrapper context */
 typedef struct H5VL_async_wrap_ctx_t {
     hid_t                under_vol_id;   /* VOL ID for under VOL */
-    void *               under_wrap_ctx; /* Object wrapping context for under VOL */
+    void                *under_wrap_ctx; /* Object wrapping context for under VOL */
```

Comment:
You need to use the https://github.com/hpc-io/vol-async/blob/develop/.clang-format file when you run clang-format command, e.g. under vol-async/src do "clang-format -i -style=file ./*.c"